portal-web/test/functional/com/liferay/portalweb/paths/pathlib/uisections/CKEditor.path

```diff
@@ -153,6 +158,17 @@
 	<td>//div[label[contains(.,'Video')]]//input</td>
 	<td></td>
 </tr>
+<!--SOURCE_CODE_PREVIEW-->
+<tr>
+	<td>CODE_PREVIEW_IFRAME</td>
+	<td>//*[contains(@class,'code_preview')]//iframe</td>
+	<td></td>
+</tr>
+<tr>
+	<td>SOURCE_CODE_DIALOG</td>
+	<td>//*[contains(@class,'cke_dialog_contents')]//*[contains(@class,'CodeMirror')]</td>
+	<td></td>
+</tr>
```

Comment:
I thought I commented on this before...must've didn't hit send. I'm not sure what this path is trying to hit because there's multiple possible elements that the locator hits.

Reply:
Multiple element patterns match:
![screenshot_17](https://user-images.githubusercontent.com/35235266/93626438-453a1100-f998-11ea-8227-fdaef3436f7f.png)

Would maybe something like this work? I think this would be a more targeted path. And maybe rename to path name to `SOURCE_CODE_TEXTAREA`?

`//*[contains(@class,'cke_dialog_contents')]//*[contains(@class,'CodeMirror')]//textarea`

Reply:
iirc, there was only one element for this locator before. I will try to use another path.